service_taxi/src/main/kotlin/org/thechance/service_taxi/domain/usecase/GetTaxiByIdUseCase.kt

```diff
@@ -0,0 +1,7 @@
+package org.thechance.service_taxi.domain.usecase
+
+import org.thechance.service_taxi.domain.entity.Taxi
+
+interface GetTaxiByIdUseCase {
```

Comment:
Same here in `ById`

Reply:
In the future we could have get taxi by car number